### PR TITLE
Handle remove user from channel

### PR DESCRIPTION
**Pull request by rehanjaved237**

### CODE REVIEW

**Comment by stakater-user** [comment]
@rehanjaved237 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/slack-operator/commit/7421e6765824d2cbba05f5a83f7eca452c23857b/checks) has Failed!

**Comment by stakater-user** [comment]
@rehanjaved237 Image is available for testing. `docker pull stakater/slack-operator:SNAPSHOT-PR-18-b8f0f184`

**Review by ahmedwaleedmalik — APPROVED** [review]
LGTM

**Comment by stakater-user** [comment]
@rehanjaved237 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/slack-operator/commit/800b710290823aef7516c17e7b54f7d37d158af9/checks) has Failed!

**Comment by stakater-user** [comment]
@rehanjaved237 Image is available for testing. `docker pull stakater/slack-operator:SNAPSHOT-PR-18-fbd150d5`

File: api/v1alpha1/channel_types.go [hunk 1]
```diff
@@ -24,12 +24,16 @@ import (
 // ChannelSpec defines the desired state of Channel
 type ChannelSpec struct {
 	// Name of the slack channel
+	// +required
 	Name string `json:"name"`
 
 	// Make the channel private or public
+	// +required
 	Private bool `json:"private,omitempty"`
```
Comment (ahmedwaleedmalik) [inline api/v1alpha1/channel_types.go:32]: This will be optional because if a user doesn't specify it's value, it will be false(public) by default

**Comment by stakater-user** [comment]
@rehanjaved237 Image is available for testing. `docker pull stakater/slack-operator:SNAPSHOT-PR-18-fb57cb87`

**Comment by stakater-user** [comment]
@rehanjaved237 Image is available for testing. `docker pull stakater/slack-operator:SNAPSHOT-PR-18-4db3e593`

**Review by ahmedwaleedmalik — APPROVED** [review]
Good job @rehanjaved237 (y)

**Pull request merged**